### ISSUE TITLE
Disabled decision buttons should show color highlights

### ISSUE THREAD

**Issue by hudson-newey** [ecoacoustics/web-components#126]
At the moment we set the `disabled` attribute on the the decision buttons if the spectrograms are loading.

However, in our theming, we currently apply a grayscale filter to every disable input element.

This means that when viewing history in the verification grid, the decision button highlights cannot be seen until all the spectrograms have been loaded.

This is incorrect, and users should be able to see what decisions are applied to a page before all the spectrograms have loaded.

### Additional note

We will have to change our theming to find a different way to visually convey that a element is disabled

**Comment by atruskie** [comment]
Yeah so the main thing here, is this makes our UI feel slow, because the grid tiles render, with their classifier colors showing, and then the decision elements finally un-disable when the spectrogram all load seconds later. It feels inconsistent.

If disabled state is down with de-saturate filter we can show disabled state clearly while still showing muted colors.
